app/core/vueRouter.coco.js

```diff
@@ -47,7 +47,8 @@ export default function getVueRouter () {
             { path: '', component: () => import(/* webpackChunkName: "teachers" */ 'app/views/school-administrator/teachers/SchoolAdminTeacherListView') },
             { path: 'teacher/:teacherId', component: () => import(/* webpackChunkName: "teachers" */ 'app/views/school-administrator/dashboard/SchoolAdminDashboardTeacherView') },
             { path: 'teacher/:teacherId/classroom/:classroomId', component: () => import(/* webpackChunkName: "teachers" */ 'app/views/courses/TeacherClassViewV2.vue') },
-            { path: 'teacher/:teacherId/classroom/:classroomId/:studentId', component: () => import(/* webpackChunkName: "teachers" */ 'app/views/teachers/classes/TeacherStudentView.vue') }
+            { path: 'teacher/:teacherId/classroom/:classroomId/:studentId', component: () => import(/* webpackChunkName: "teachers" */ 'app/views/teachers/classes/TeacherStudentView.vue') },
+            { path: 'licenses/stats', component: () => import(/* webpackChunkName: 'prepaids' */ 'app/views/school-administrator/dashboard/LicenseTableView.vue') }
```

Comment:
better to have chunkName like licensesStats instead of prepaids as prepaids can be a chunk of vue prepaid store if it ever split like that